SR700_Artisan_Server/Roaster_Cooling.py

```diff
@@ -12,6 +12,14 @@ def main():
     roast_control.enable_fan_manual_mode()
     roast_control.set_fan_speed(9)
     roast_control.run_cooling()
+
+    # if the Artisan alarms trigger before detecting manual fan mode, then 
+    # they will reset the fan to a (likely) lower speed.  So, keep 
+    # keep setting fan to recover from race condition.
+    time.sleep(2)
+    roast_control.set_fan_speed(9)
+    time.sleep(2)
```

Comment:
very interesting, I noticed before something was not correct. This should fix it

Reply:
Yeah I further had to add a bit more delay to trigger the fan over the course of 10 seconds or so.